tests/pci_passthrough/test_pci_passthrough.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+import logging
+
+# Requirements:
+# - an XCP-ng host (--hosts) >= 8.3 with a PGPU to hide from dom0
+
+XEN_CMDLINE = '/opt/xensource/libexec/xen-cmdline'
+
+@pytest.mark.reboot # reboots the host
+@pytest.mark.incremental # tests depend on each other. If one test fails, don't execute the others
+class TestPCIPassthrough:
+    def test_disable_dom0_access(self, host, enabled_pci_uuid):
+        host.xe("pci-disable-dom0-access", {"uuid": enabled_pci_uuid})
+        assert host.xe("pci-get-dom0-access-status", {"uuid": enabled_pci_uuid}) == "disable_on_reboot"
+
```

Comment:
On some other tests, you don't add a newline here, so I think it's not necessary.
```suggestion
```